### DISCUSSION TITLE
Request for guidance

### DISCUSSION THREAD

**Discussion by animeshk923** [outreachy/website#519] [Q&A] [answered]
Greetings!

I just found a grammatical error on the website while reading documentation about blogging FAQ.

How do I open an issue to address this?

**Comment by SaxenaAnushka102** [comment, marked as answer]
Hi @AnimeshKumar923! You can refer to the [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) and then [create an issue](https://github.com/outreachy/website/issues/new). Make sure to follow the [Code of Conduct](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). All the best!
